install/1.44-1.45/update.sql

```diff
@@ -16,11 +16,11 @@ CREATE TABLE IF NOT EXISTS `wD_MissedTurns` (
 	  KEY `missedPerUserPerDate` (`userID`,`turnDateTime`)
 	) ENGINE=InnoDB DEFAULT CHARSET=utf8;
 	
-ALTER TABLE `wD_Members` ADD `excusedMissedTurns` int(10) unsigned;
-ALTER TABLE `wD_Games` ADD `excusedMissedTurns` int(10) unsigned;
-ALTER TABLE `wD_Users` ADD `yearlyMissedTurnCount` mediumint(8) unsigned;
-ALTER TABLE `wD_Users` ADD `shortTermMissedTurnCount` mediumint(8) unsigned;
-ALTER TABLE `wD_Users` ADD `yearlyPhaseCount` mediumint(8) unsigned;     
+ALTER TABLE `wD_Members` ADD `excusedMissedTurns` int(10) unsigned DEFAULT 0;
```

Comment:
Oh this is interesting, I had not considered that this is going to impact ongoing games when we do this change. Do you think we should default existing games to 1 or 2 excuses to "simulate" being able to NMR twice currently before being forced into CD like the sites currently do?

Reply:
I have chosen 2 as default in game creation and would go with 2 here, as well. With 2 excuses you will pushed into CD after the third NMR, which seems to be a good middle way between the current situation of CD after 2 consecutive NMRs and infinite not consecutive ones.